src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -90,6 +89,11 @@ internal OpenTelemetryLoggerProvider(OpenTelemetryLoggerOptions options)
             }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether or not formatted messages should be included on log messages.
+        /// </summary>
+        public bool IncludeFormattedMessage { get; }
```

Comment:
Bugging you again with the "we sure we wanna expose this publicly, yet?" question 😆... I don't have a strong feeling that we shouldn't, just wanna raise it for discussion.

Brainstorming alternatives, would it make sense instead to expose this on the `LogEmitter` class?

At the moment, both the extensions are instantiated with an instance of the logging provider and then
1. store the `IncludeFormattedMessage` setting, then
2. call `CreateEmitter()`

Would it make sense to just instantiate the extensions with an emitter instead of the provider? So like Serilog would look like

```csharp
Log.Logger = new LoggerConfiguration()
    .WriteTo.OpenTelemetry(openTelemetryLoggerProvider.CreateEmitter()) // <- Register OpenTelemetry Serilog sink
    .CreateLogger();
```

Again, I don't have a strong sense that one option is better than the other at this point, just curious of your thoughts.